reactor-core/src/main/java/reactor/core/publisher/UnicastProcessor.java

```diff
@@ -35,10 +35,47 @@
 
 /**
  * A Processor implementation that takes a custom queue and allows
- * only a single subscriber.
+ * only a single subscriber. UnicastProcessor allows demultiplexing of the events which
+ * means that that Processors support multiple producers and only one consumer.
  *
  * <p>
- * The implementation keeps the order of signals.
+ *      <img width="640" src="https://raw.githubusercontent.com/reactor/reactor-core/master/src/docs/marble/unicastprocessornormal.png" alt="">
+ * </p>
+ *
+ * </br>
+ * </br>
+ *
+ * <p>
+ *      <b>Note: </b> UnicastProcessor does not respect of actual subscriber's
+ *      demand as it is described in
+ *      <a href="http://www.reactive-streams.org/">Reactive Streams Spec</a>. However,
+ *      Unicast Processor embraces configurable Queue internally which allows enabling
+ *      backpressure support and preventing of consumer's overwhelming.
+ *
+ *      Hence, interaction model between Producers and UnicastProcessor will be PUSH
+ *      only and between UnicastProcess and Consumer PUSH-PULL as defined in
```

Comment:
`UnicastProcess` -> `UnicastProcessor`. Split in 2 sentences, second one for UnicastProcessor to Consumer side. Maybe avoid uppercase on `Producers` and `Consumer` to clarify these are not class names.